src/main/kotlin/com/terraformation/backend/customer/db/UserStore.kt

```diff
@@ -178,6 +179,26 @@ class UserStore(
     return user?.let { rowToIndividualUser(it) }
   }
 
+  /**
+   * Returns the row for the user with a given email address.
+   *
+   * @param email Email address to search for. If the address has upper-case characters, they will
+   *   be folded to lower case.
+   * @return null if no user has the requested email address, or if users that do have been deleted.
+   */
+  fun fetchUserRowByEmail(email: String): UsersRow? {
```

Comment:
Nit: This should probably be `fetchUsersRowByEmail` (plural "Users") to match the type name.

Reply:
Thanks. I'm renaming it now to match the return type per your later comment